From c470f32fc91789c0031f7352d34a5a6ac3c864b3 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Thu, 15 Nov 2018 19:20:12 -0700
Subject: [PATCH 4/6] klte-common: restorecon I/O scheduler tunables before
 touching them

* early-init is before the global restorecon of /sys gets run, so
  we need to set a context on these before changing attributes

Change-Id: I493a9bb0c1bd2ea83a815e07169b8b689cf76a5d
---
 rootdir/etc/init.qcom.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rootdir/etc/init.qcom.rc b/rootdir/etc/init.qcom.rc
index 58ff196..93fd5aa 100644
--- a/rootdir/etc/init.qcom.rc
+++ b/rootdir/etc/init.qcom.rc
@@ -33,6 +33,7 @@ on early-init
     chmod 0755 /sys/kernel/debug
 
     # Increase max readahead size to 512 KB
+    restorecon --recursive /sys/block/mmcblk0/queue
     write /sys/block/mmcblk0/queue/read_ahead_kb 512
     write /sys/block/mmcblk0/queue/scheduler noop
 
-- 
2.17.1

